ui/app/components/clients/page/overview.hbs

```diff
@@ -5,27 +5,24 @@
 
 <Clients::RunningTotal
   @isSecretsSyncActivated={{this.flags.secretsSyncIsActivated}}
-  @byMonthActivityData={{this.byMonthActivityData}}
+  @byMonthNewClients={{this.byMonthNewClients}}
   @isHistoricalMonth={{and (not this.isCurrentMonth) (not this.isDateRange)}}
```

Comment:
Can we remove any logic for `this.isCurrentMonth` since that's no longer query-able? Can definitely happen in a follow-on PR, but could you make a ticket to tackle it? Anything client counts requires so much context, it's important to make sure we promptly clean things up as much as possible as it's very easy to leave outdated logic behind.